### PR TITLE
Update scroll-prover-sdk commit hash to latest commit

### PR DESCRIPTION
**Pull request by Roee-87**
This PR updates the import for the `scroll-proving-sdk` to use the latest commit hash.

### CODE REVIEW

**Review by katiemckeon — COMMENTED** [review]
Build works! But running the code alone (for development) will be limited if we merge this as-is.  When I do `cargo run --release`, the executable crashes with the following output.  I'm thinking we've either got to put some shim in the code for a standalone development or if no code revisions are necessary, mark up our development section in the readme with the solution
```
2024-10-30T18:13:25.405956Z  INFO sindri_scroll_sdk: [Sindri client]: "https://sindri.app/api/v1/circuit/scroll-tech/batch_prover:v0.13.1/detail"
2024-10-30T18:13:25.621811Z  INFO sindri_scroll_sdk: [Sindri client], detail, received response
2024-10-30T18:13:25.722349Z  INFO scroll_proving_sdk::coordinator_handler::api: [coordinator client], /coordinator/v1/login, sent request
2024-10-30T18:13:25.737818Z  INFO scroll_proving_sdk::coordinator_handler::api: [coordinator client], /coordinator/v1/login, received response
thread 'main' panicked at /home/ubuntu/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hyper-0.14.28/src/server/server.rs:81:13:
error binding to 0.0.0.0:80: error creating server listener: Permission denied (os error 13)
```

**Review by katiemckeon — COMMENTED** [review]
I investigated further, if your `config.json` supplies no health check address then the default is `0.0.0.0.:80` and generally apps cant bind to port 80 unless they've got the root permissions.  So a way to fix this for dev is to just update your config.json and supply practically any port that's not 80. 

Recommendation: update the `example.config.json` file and add something like this:
```
    "health_listener_addr": "0.0.0.0:5678"
```

**Comment by Roee-87** [comment]
I added in a dummy address in the example.config.json in this commit:  [13c7860](https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/20/commits/13c7860b6fa1593351b0c59fa15418e11c254f07)

File: example.config.json [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
-        "endpoint": "https://your-l2-rpc"
+        "endpoint": "https://l2-rpc:8545"
     },
     "prover": {
         "circuit_type": 3,
         "circuit_version": "v0.13.1",
         "n_workers": 1,
         "cloud": {
-            "base_url": "https://sindri.app",
+            "base_url": "https://sindri.app/api/v1/",
```
Comment (katiemckeon) [inline example.config.json:18]: This revision won't work. It was purposely changed in https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/15

I suspect you are working from an older version of main and need to pull in the latest remote changes
```suggestion
            "base_url": "https://sindri.app",
```

File: example.config.json [hunk 2]
```diff
@@ -2,24 +2,25 @@
     "prover_name_prefix": "sindri_",
     "keys_dir": "keys",
     "coordinator": {
-        "base_url": "https://your-coordinator-api",
+        "base_url": "https://coordinator-ap:80",
```
Comment (katiemckeon) [inline example.config.json:5]: typo here
```suggestion
        "base_url": "https://coordinator-api:80",
```

Comment (Roee-87) [reply example.config.json]: Done:  [0d5979d](https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/20/commits/0d5979d0d9b98b93ea8939543c241ba341623b8c)

File: example.config.json (hunk 1 above)
Comment (Roee-87) [reply example.config.json]: Done:  [1845dc2](https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/20/commits/1845dc24e37aa5c342d80b0165806f5bbd95d423)

**Review by katiemckeon — APPROVED** [review]
LGTM!. Pending CI test, feel free to merge

**Pull request merged**